popup.js

```diff
@@ -36,6 +36,17 @@ $("#ERPLoginID").blur(function() {
   }
 });
 
+
+function show_open_if_already_saved() {
+  chrome.storage.sync.get(results => {
+    // chrome.extension.getBackgroundPage().console.log("results are  ", results);
```

Comment:
this comment can be removed right?

Reply:
Yeah, I have deleted it now. Thanks.

By the way, why is it preferred to remove code rather than commenting it out in a case like this?

Reply:
when i added the function `show_open_if_already_saved` i put that line for debugging purpose, but hadn't removed it
it was my fault actually

i realized after i ran [git blame](https://github.com/metakgp/erp-auto-login/blame/6594487a5bfe11d65784d9dab52f6f4cd0a8186e/popup.js#L76) and checked that

Reply:
> why is it preferred to remove code rather than commenting it out in a case like this?

This is a beautiful question Sahil. In a concise manner, the reason is that it can be confusing to others who work on the project as to why the piece of code is commented out (this is called noise).

Please read more on this here, https://www.nayuki.io/page/dont-share-commented-out-code